src/frontend/src/lib/components/ui/ThreeBackground.svelte

```diff
@@ -152,8 +152,18 @@ void main(){
 		camera.aspect = container.clientWidth / container.clientHeight;
 		camera.updateProjectionMatrix();
 	};
+
+	const resizeAction = (node: HTMLElement) => {
+		const resizeObserver = new ResizeObserver(handleResize);
```

Comment:
Should the callback be debounced? e.g. if you resize your browser, isn't the obserser triggers lots of time and therefore consumes quite some memory? Or it's fine to resize often?

Reply:
Sorry, there was a simpler way, please check last commit